schematron/v2.1.0/L100_Audit.csv

```diff
@@ -0,0 +1,75 @@
+phase title,phase see,pattern title,pattern see,rule title,rule context,assert test,assert description,assert severity,notes
+Facility Description,ASHRAE 211 6.1.1,Misc Building Info,ASHRAE 211 6.1.1.1 and 6.1.1.2,Basic Building Info,/auc:BuildingSync/auc:Facilities/auc:Facility/auc:Sites/auc:Site/auc:Buildings/auc:Building,auc:PremisesName,,,
+,,,,,,auc:Address/auc:City,,,
+,,,,,,auc:Address/auc:State,,,
+,,,,,,auc:Address/auc:PostalCode,,,
+,,,,,,auc:Address/auc:StreetAddressDetail/auc:Simplified/auc:StreetAddress,,,
+,,,,,,auc:FloorsAboveGrade,,,
+,,,,,,auc:FloorsBelowGrade,,,
+,,,,,,auc:FloorAreas/auc:FloorArea[auc:FloorAreaType/text() = 'Gross'],,,
+,,,,,,auc:FloorAreas/auc:FloorArea[auc:FloorAreaType/text() = 'Conditioned'],,,
+,,,,,,auc:BuildingClassification,,,
+,,,,,,auc:OccupancyClassification,,,
+,,,,,,auc:YearOfConstruction,,,
+,,,,,,auc:YearOfLastMajorRemodel,,WARNING,
+,,,,,,auc:YearOfLastEnergyAudit,,WARNING,
+,,,,,,(auc:BuildingClassification/text() != 'Mixed use commercial' and auc:BuildingClassification/text() != 'Residential') or auc:SpatialUnits/auc:SpatialUnit[auc:SpatialUnitType/text() = 'Apartment']/auc:NumberOfUnits,"If BuildingClassificatoin implies residents (Mixed use commercial or Residential), number of apartments units must be defined at auc:SpatialUnits/auc:SpatialUnit[auc:SpatialUnitType = 'Apartment]/auc:NumberOfUnits.",,

```

Comment:
Don't we want it to be:
- `auc:BuildingClassification/text() == 'Mixed use commercial' or auc:BuildingClassification/text() == 'Residential'`?

Reply:
I think test should accomplish: "If the classification is Mixed use commercial or Residential, then there there must be a spacial unit with number of units". In other words, this assertion should pass if:
(1) it's not either of those classification types, or (2) if it has apartment units (implied that it must be Mixed use commercial or Residential).

Reply:
Good catch! I've updated it and added tests for this